pkg/interfaces/contracts/liquidity-mining/ILiquidityGauge.sol

```diff
@@ -28,4 +32,12 @@ interface ILiquidityGauge {
     function killGauge() external;
 
     function unkillGauge() external;
+
+    function set_max_relative_weight(uint256 maxRelativeWeight) external;
```

Comment:
I'd consider something like `set_relative_weight_cap`, to use consistent language everywhere.

Reply:
about naming: I thought `max_relative_weight` and `capped_relative_weight` as two different things. The former refers to the limit / upper bound itself, and the latter refers to the upper bound _applied_ to the value you get from the controller.

We can use `set_relative_weight_cap` here, but `get_relative_weight_cap` would then be a getter for the upper bound, not the bound applied to the weight (i.e. `min(upper_bound, controller.gauge_relative_weight(...)`).

If we use `cap` everywhere, do you still think it's clear?

Reply:
There are two concepts: the limit itself (which can be read and set), and the weight with the limit applied. We should use a similar term for both, e.g. get/set limit and get limited weight, or get/set cap and get capped weight.

You could go with get/set max weight and get maxed weight, but that's a bit odd. I personally like capped.

Reply:
thanks; done.